### DISCUSSION TITLE
Example Project Build Artifacts and Release Actions

### DISCUSSION THREAD

**Discussion by jumper385** [zephyrproject-rtos/zephyr#63008] [Q&A]
Hi All, 

I'm just wondering if there are plans to add build artefacts outputs (zephyr.hex file outputs for each compiled binary) to the example project's build action? Additionally, are there discussions to automate a "release" with such artefacts attached to them?

Kind Regards,
Henry Chen

**Comment by nordicjm** [comment]
I don't see why where would be, it's a dummy application
